### ISSUE TITLE
Can't get example to display for object array

### ISSUE THREAD

**Issue by wyfiwyfi** [knuckleswtf/scribe#328]
_This is a restatement of [Issue #323](https://github.com/knuckleswtf/scribe/issues/323), which I've closed. That original issue was created in an incorrect format and without the bug tag._

- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.knuckles.wtf/laravel/troubleshooting).** Yes, I have looked at this and applied the relevant suggestions.

**What happened?**
This should be straightforward, but after combing through the documentation and other issues, I can't get the example for an array of objects to work. 

What I want to produce in the (PHP json) documentation example is something like this:
```
...
'json' => [
            'geopoints' => [
                [
                    'latitude' => 23.61721882,
                    'longitude' => 67.43423423,
                ],
                [
                    'latitude' => 66.12345432,
                    'longitude' => 90.98762342,
                ],
            ],
        ],
...

```
That is, an array of objects with two (preferably three) examples of objects using 8-decimal place numbers that I provide.

Here is what I think should work:
```
* @bodyParam geopoints object[] List of latitude and longitude pairs Example: [{"latitude": 23.61721882, "longitude": 67.43423423}, {"latitude": 66.12345432, "longitude": 90.98762342}]
* @bodyParam geopoints[].latitude number required Number between -90 and 90 and up to 8 decimals. Must have corresponding longitude.
* @bodyParam geopoints[].longitude number required Number between -180 and 180 with up to 8 decimals. Must have corresponding latitude.
```

But any example I include in the top level object is ignored and replaced by an auto-generated example. The above produces this (correct format, but not the numbers given):
```
...
'json' => [
            'geopoints' => [
               [
                    'latitude' => 1394.7993,
                    'longitude' => 34.90597,
                ],
                [
                    'latitude' => 1394.7993,
                    'longitude' => 34.90597,
                ],
            ],
        ],
...

```
If I instead place the examples on the item lines, like this:
```
* @bodyParam geopoints object[] List of latitude and longitude pairs
* @bodyParam geopoints[].latitude number required Number between -90 and 90 and up to 8 decimals. Must have corresponding longitude. Example: 23.61721882
* @bodyParam geopoints[].longitude number required Number between -180 and 180 with up to 8 decimals. Must have corresponding latitude. Example: 67.43423423
```

the result is:
```
...
'json' => [
            'geopoints' => [
                [
                    'latitude' => 23.61721882,
                    'longitude' => 67.43423423,
                ],
                [
                    'latitude' => 23.61721882,
                    'longitude' => 67.43423423,
                ],
            ],
        ],
...
```
This is better, since the structure is correct and the one set of numbers I entered as example are used, but I can't figure out a way to produce multiple geopoints (three or more would be optimal) with numbers that I provide. 

Even though it's not documented, I've tried arrays on the item lines, like the following, but with poor results (the examples generated in the documentation are all zeroes):

```
* @bodyParam geopoints object[] List of latitude and longitude pairs
* @bodyParam geopoints[].latitude number required Number between -90 and 90 and up to 8 decimals. Must have corresponding longitude. Example: [{23.61721882, 66.12345432}]
* @bodyParam geopoints[].longitude number required Number between -180 and 180 with up to 8 decimals. Must have corresponding latitude. Example: [{67.43423423, 90.98762342}]
```

I've experimented with some other ideas, but can't seem to get the result I'm after. Can someone spot what I'm missing? 

**Screenshots and stack traces:**

Hi - I'm using Scribe 3.10.2/Lumen 6.3.3/PHP 8.0.5 /Ubuntu 20 on Homestead.
**My environment:**
 - PHP version: 8.0.5 
 - Framework (Laravel/Lumen): Lumen
 - Laravel/Lumen version (from `composer show laravel/framework` or `composer show laravel/lumen-framework`): Lumen 6.3.5
 - Scribe version (from `composer show knuckleswtf/scribe`):  3.10.2

**My Scribe config (minus the comments):**
```
use Knuckles\Scribe\Extracting\Strategies;

return [

    'theme' => 'default',
    'title' => null,
    'description' => '',
    'base_url' => null,
    'routes' => [[
        'match' => [
            'domains' => ['*'],
            'prefixes' => ['*'],
            'versions' => ['v1'],
        ],
        'include' => [],
        'exclude' => [],
        'apply' => [
        'headers' => ['Cache-Control' => 'no-cache', 'Content-Type' => 'application/json', 'Accept' => '*/*'],
        'response_calls' => [
        'methods' => ['GET', 'POST', 'PATCH', 'PUT'],
        'config' => ['app.env' => 'documentation'],
        'cookies' => [],
        'queryParams' => [],
        'bodyParams' => [],
        'fileParams' => [],
            ],
        ],
    ]],
    'type' => 'static',
    'static' => [
        'output_path' => 'public/docs',
    ],
    'laravel' => [
        'add_routes' => true,
        'docs_url' => '/docs',
        'middleware' => [],
    ],
    'try_it_out' => [
        'enabled' => false,
        'base_url' => null,
        'use_csrf' => false,
        'csrf_url' => '/sanctum/csrf-cookie',
        'use_csrf' => false,
        'csrf_url' => '/sanctum/csrf-cookie',
    ],
    'auth' => [
        'enabled' => true,
        'default' => true,
        'in' => 'header',
        'name' => 'Api-token',
        'use_value' => env('API_TOKEN'),
        'placeholder' => '{YOUR_AUTH_KEY}',
        'extra_info' => '',
    ],
    'intro_text' => <<<INTRO
Welcome to the Ekofolio.com API documentation!

<aside>As you scroll, you'll see documentation and code examples for working with the API. Endpoints are grouped by function and appear in the dark pane to the left. Clicking on a group will expand it and show its endpoints. Then, clicking on an endpoint will show tis documentation in the larger pane to the right. </aside>
INTRO

    ,
    'example_languages' => [
        'bash',
        'javascript',
        'php',
    ],
    'postman' => [
        'enabled' => true,
        'overrides' => [
            // 'info.version' => '2.0.0',
        ],
    ],
    'openapi' => [
        'enabled' => false,

        'overrides' => [
            // 'info.version' => '2.0.0',
        ],
    ],
    'default_group' => 'Misc Endpoints',
    'logo' => false,
    'faker_seed' => null,
    'strategies' => [
        'metadata' => [
            Strategies\Metadata\GetFromDocBlocks::class,
        ],
        'urlParameters' => [
            Strategies\UrlParameters\GetFromLaravelAPI::class,
            Strategies\UrlParameters\GetFromLumenAPI::class,
            Strategies\UrlParameters\GetFromUrlParamTag::class,
        ],
        'queryParameters' => [
            Strategies\QueryParameters\GetFromFormRequest::class,
            Strategies\QueryParameters\GetFromInlineValidator::class,
            Strategies\QueryParameters\GetFromQueryParamTag::class,
        ],
        'headers' => [
            Strategies\Headers\GetFromRouteRules::class,
            Strategies\Headers\GetFromHeaderTag::class,
        ],
        'bodyParameters' => [
            Strategies\BodyParameters\GetFromFormRequest::class,
            Strategies\BodyParameters\GetFromInlineValidator::class,
            Strategies\BodyParameters\GetFromBodyParamTag::class,
        ],
        'responses' => [
            Strategies\Responses\UseTransformerTags::class,
            Strategies\Responses\UseResponseTag::class,
            Strategies\Responses\UseResponseFileTag::class,
            Strategies\Responses\UseApiResourceTags::class,
            Strategies\Responses\ResponseCalls::class,
        ],
        'responseFields' => [
            Strategies\ResponseFields\GetFromResponseFieldTag::class,
        ],
    ],
    'fractal' => [
        'serializer' => null,
    ],
    'routeMatcher' => \Knuckles\Scribe\Matching\RouteMatcher::class,
    'database_connections_to_transact' => [config('database.default')]
];

```

**Additional info:**

**Comment by shalvah** [comment]
I'm currently changing some things around example generation for arrays. The next version of Scribe should only generate one item in an array, and you should be able to override that example completely by specifying your array on the parent (allowing you to specify 15 items if you want). Should be implemented today or tomorrow.

**Comment by shalvah** [comment]
Should be fixed in 3.11.0 👍 Scribe will only generate 1-item arrays by default. You can override the item in the array by specifying at the field level, or override the entire example by specifying at the parent level.
